src/Suave.Xsp/Library.fs

```diff
@@ -0,0 +1,178 @@
+﻿namespace Suave
+
+open System
+open System.IO
+open System.Web
+open System.Web.Hosting
+open System.Text
+
+open Suave
+open Suave.Types
+open Suave.Types.Codes
+open Suave.Http
+open Suave.Utils
+
+open Suave.Sockets
+
+module Xsp =
```

Comment:
Could you add a short summary comment on each module and type definition, and preferably each member too?  For example what Xsp stands for  (I know it's going to be eXtreme Server Pages :) ) thanks